modules/core/src/test/java/org/apache/ignite/internal/processors/cache/transactions/AtomicOperationsInTxTest.java

```diff
@@ -58,122 +61,186 @@ public class AtomicOperationsInTxTest extends GridCommonAbstractTest {
     /** {@inheritDoc} */
     @Override protected void beforeTestsStarted() throws Exception {
         super.beforeTestsStarted();
+    }
 
+    /** {@inheritDoc} */
+    @Override protected void beforeTest() throws Exception {
+        super.beforeTest();
         startGrid(0);
     }
 
+    /** {@inheritDoc} */
+    @Override protected void afterTest() throws Exception {
+        super.afterTest();
+        stopGrid(0);
+    }
+
     /**
+     * Tests whether enabling atomic cache operations within transactions is allowed.
+     * Since 2.15.0 the default behaviour is set to {@code false}.
```

Comment:
No need to mention about 2.15 in those tests.

Reply:
Deleted mentioning, thank you

Reply:
Deleted mentioning, thank you